changelogs/fragments/8439-feature-keep_keys.yml

```diff
@@ -0,0 +1,2 @@
+major_changes:
+  - New filter keep_keys. https://github.com/ansible-collections/community.general/pull/8439
```

Comment:
Please remove the changelog fragment; new plugins are automatically announced in the changelog due to their top-level `version_added` in the documentation.